Remove obsolete workarounds for previous Solaris problems with these
functions and C++ programs.

--- a/Source/WTF/wtf/MathExtras.h	2018-02-14 09:17:11.178105324 -0800
+++ b/Source/WTF/wtf/MathExtras.h	2018-02-14 09:17:48.800502215 -0800
@@ -34,10 +34,6 @@
 #include <stdlib.h>
 #include <wtf/StdLibExtras.h>
 
-#if OS(SOLARIS)
-#include <ieeefp.h>
-#endif
-
 #if OS(OPENBSD)
 #include <sys/types.h>
 #include <machine/ieee.h>
@@ -75,24 +71,6 @@
 const float sqrtOfTwoFloat = static_cast<float>(M_SQRT2);
 #endif
 
-#if OS(SOLARIS)
-
-namespace std {
-
-#ifndef isfinite
-inline bool isfinite(double x) { return finite(x) && !isnand(x); }
-#endif
-#ifndef signbit
-inline bool signbit(double x) { return copysign(1.0, x) < 0; }
-#endif
-#ifndef isinf
-inline bool isinf(double x) { return !finite(x) && !isnand(x); }
-#endif
-
-} // namespace std
-
-#endif
-
 #if COMPILER(MSVC)
 
 // Work around a bug in Win, where atan2(+-infinity, +-infinity) yields NaN instead of specific values.
